src/activities/drawletters/drawletters.qrc~

```diff
@@ -0,0 +1,48 @@
+<RCC>
```

Comment:
Hi,
I can't test for now, I'll test as soon as I can. Globally, the code seems good.
Johnny